django_celery_beat/utils.py

```diff
@@ -11,6 +17,68 @@
 # see Issue #222
 now_localtime = getattr(timezone, 'template_localtime', timezone.localtime)
 
+logger = get_logger(__name__)
+
+
+def generate_keys(
```

Comment:
I am not sure why we need rsa key?

Reply:
it's need to sign serialized objects so as not to execute any python code that may appear in the database (execute only signed serilized code which can be signed and added only from main proc)

Reply:
https://davidhamann.de/2020/04/05/exploiting-python-pickle/

Reply:
why noy json serializer? it's default in celery

Reply:
JSON serializer should be the default task serializer in the integration packages as well

Reply:
okay, I'll rewrite

Reply:
dill is ok for pickle, but pickle is not the default serializer. so rewriting this with json would be great